Codigo/mdlCOmercioConUsuario.bas

```diff
@@ -271,38 +271,52 @@ Public Sub AceptarComercioUsu(ByVal UserIndex As Integer)
         'Por si las moscas...
 158     If TerminarAhora Then GoTo FinalizarComercio
     
+        Dim registroComercio As String
+        registroComercio = "Usuario " & UserList(UserIndex).name & " comercio con " & UserList(OtroUserIndex).name & "."
```

Comment:
use english `User` instead of `Usuario` abd `Traded with` instead of ` comercio con`